### PR TITLE
Clean up search-replace and apply-setters docs

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR is to clean-up the docs for search-replace and apply-setters. The docs are written for invoking functions with kpt v1 binary.

### CODE REVIEW

File: examples/mutators/apply-setters/simple/README.md [hunk 1]
```diff
@@ -55,17 +55,17 @@ kind: MyKind
...
 
 ## Function invocation
 
 Get the config example and try it out by running the following commands:
 
 ```sh
-kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple .
-kpt fn run simple
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple
+$ kpt fn eval simple/ --image gcr.io/kpt-fn/apply-setters:unstable --fn-config simple/fn-config.yaml
```
Comment (frankfarzan) [inline examples/mutators/apply-setters/simple/README.md:68]: @mengqiy This falls under https://github.com/GoogleContainerTools/kpt/issues/1548

What's the general approach here, I expected this migration to be done holistically, not piecemeal.

Comment (phanimarupaka) [reply examples/mutators/apply-setters/simple/README.md:68]: Can we unblock this PR.

Comment (mengqiy) [reply examples/mutators/apply-setters/simple/README.md:68]: We should do the migration all at once when we approaching CLI v1.0 which is weeks away from now.

@phanimarupaka Can we keep using `kpt fn run`? Are there any issues?

File: examples/mutators/apply-setters/simple/README.md [hunk 2]
```diff
@@ -55,17 +55,17 @@ kind: MyKind
...
 ```
 
 ## Function invocation
 
 Get the config example and try it out by running the following commands:
 
 ```sh
-kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple .
-kpt fn run simple
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/mutators/apply-setters/simple
```
Comment (mengqiy) [inline examples/mutators/apply-setters/simple/README.md:67]: This path needs to be updated. `mutators` and `validators` directories are gone.

File: examples/mutators/apply-setters/simple/README.md (hunk 1 above)
Comment (phanimarupaka) [reply examples/mutators/apply-setters/simple/README.md]: Done.

File: examples/mutators/apply-setters/simple/README.md (hunk 2 above)
Comment (phanimarupaka) [reply examples/mutators/apply-setters/simple/README.md]: Done.

File: functions/go/search-replace/README.md [hunk 3]
```diff
@@ -176,4 +144,60 @@ a:
...
+# Matches fields with name "containerPort" arbitrarily deep in "spec" that have value of 80:
+$ kpt fn run --image gcr.io/kpt-fn/search-replace:unstable -- by-path='spec.**.containerPort' by-value=80
+```
+
+```sh
+# Set namespaces for all resources to "bookstore", even namespace is not set on a resource:
+$ kpt fn run --image gcr.io/kpt-fn/search-replace:unstable -- by-path='metadata.namespace' put-value='bookstore'
+```
+
+```
```
Comment (mengqiy) [inline functions/go/search-replace/README.md:176]:
```sh

**Review by mengqiy — APPROVED** [review]
one nit

**Pull request merged**